gov_uk_dashboards/assets/dashboard.css

```
@@ -12556,7 +12556,8 @@ number-* {
 
 /* This prevents the size of the menu changing when a link becomes active and therefore bold */
 .dashboard-container .moj-side-navigation {
-  width: 15%;
+  width: 20%;
+  padding-right: 2%;
```

```suggestion
  padding-inline-end: 2%;
```

should consider using these properties rather than left or right  Its called logical properties